app-backend/api/src/main/scala/annotation-project/AnnotationProjectTaskRoutes.scala

```diff
@@ -143,22 +158,27 @@ trait AnnotationProjectTaskRoutes
       } {
         entity(as[Task.TaskGridFeatureCreate]) { tgf =>
           complete(
-            StatusCodes.Created,
-            TaskDao
-              .insertTasksByGrid(
-                Task
-                  .TaskPropertiesCreate(
-                    TaskStatus.Unlabeled,
-                    projectId,
-                    None,
-                    None,
-                    None,
-                    None
-                  ),
-                tgf,
-                user
-              )
-              .transact(xa)
+            StatusCodes.Accepted,
+            taskGridBlocker
+              .blockOn(
+                TaskDao
+                  .insertTasksByGrid(
+                    Task
+                      .TaskPropertiesCreate(
+                        TaskStatus.Unlabeled,
+                        projectId,
+                        None,
+                        None,
+                        None,
+                        None
+                      ),
+                    tgf,
+                    user
+                  )
+                  .transact(xa)
+                  .start(taskGridContextShift)
```

Comment:
What is `start` doing here? This compiles if I remove it and it's a little confusing at first blush since the second argument provides the `ContextShift` to perform the IO in

Reply:
My (untested!) understanding is that akka-http will wait for the future regardless of what context shift it's running in. The context shift ensures that our background processes don't mess with the request/response cycle, and the `Fiber` returned by `start` ensures that we don't need to wait for the `Future` to complete before getting a response back. I'll test that understanding though.

Reply:
My understanding was correct!

```
❯ cat tfc.json | http --auth-type=jwt ":9100/api/annotation-projects/4ec05f41-aa1d-47e4-95a9-deeb3eaea293/tasks/grid"
HTTP/1.1 504 Gateway Time-out
...
```

Without `start`, the response has to wait on the computation to complete